Cakes-Co/index.html

```diff
@@ -1,16 +1,104 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
-</head>
+    <link rel="stylesheet" href="style.css" />
+    <link href="https://fonts.cdnfonts.com/css/birthday-2" rel="stylesheet" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Dancing+Script&family=Playfair+Display:ital@1&display=swap" rel="stylesheet">
+  </head>
 
-<body>
+  <body>
     <!-- Add your markup here -->
-</body>
 
-</html>
\ No newline at end of file
+    <header>
+      <section class="a">
+        <!-- <div> -->
+          <img
+            class="imagea"
+            src="https://images.unsplash.com/photo-1612809076670-c93d4176e065?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxleHBsb3JlLWZlZWR8OTZ8fHxlbnwwfHx8fHw%3D&auto=format&fit=crop&w=600&q=60"/>
+          <div class="flexxy">
+            <div >
+              <div class="menu"></div>
+              <div class="menu"></div>
+              <div class="menu"></div>
+            </div>
+            <div class="mylinks">
+          <a href="#home"> HOME</a>
```

Comment:
Your indentation is not fully consistent. This does not make a functional difference BUT you should use Prettier as a VSCode plugin (ideally format on every save) and then you will not need to format manually.